backend/ibutsu_server/db/models.py

```diff
@@ -164,7 +177,66 @@ class WidgetConfig(Model, ModelMixin):
     navigable = Column(Boolean, index=True)
     params = Column(mutable_json_type(dbtype=PortableJSON()))
     project_id = Column(PortableUUID(), ForeignKey("projects.id"), index=True)
```

Comment:
This is just the preparation right? This relation will be removed in a later PR?

Reply:
Yeah, I just left that in there for now. Later on we'll need to do some data migration and move those widgets to a "Default" or some other dashboard.